docs/_panels/github-backlog.md

```diff
@@ -1,20 +0,0 @@
----
-title: GitHub Backlog
-description: focused on pending (open) tasks.
-author: Bitergia
-screenshot: sigils/github-issues-backlog.png
-created_at: 
-grimoirelab_version: 0.2.0
-layout: panel
----
-
-This panel focuses on pending tasks, that is, those Issues
-that remain open (note that here we work with data retrieved in the moment of
-dashboard creation).
-
-It is possible to filter data in the same way as in other GitHub panels
-(by submitter, repository and domain).
-
-A list with the oldest Issues is also shown, providing direct
-links to them and some basic information as their titles, how long they have
-been open and when they were created.
```

Comment:
I checked that this file was renamed to ` docs/_panels/github-issues-backlog.md`. New version of the file [here](https://github.com/chaoss/grimoirelab-sigils/pull/457/files#diff-4e4408d4e8c4b339c4d7de584b2dbab7).

Reply:
Yes, this was outdated. We had documented the old dashboard that was split into two.